src/resonanceReconstruction/reichMoore/Base/src/neutronWaveNumber.hpp

```diff
@@ -0,0 +1,25 @@
+Quantity< InvRootBarns >
+neutronWaveNumber( const Quantity< ElectronVolts > energy ) const {
+  using Units4Constant =
+    decltype( pow( Barns() * ElectronVolts(), Ratio<-1,2> ) );
+
+  constexpr Quantity< Units4Constant >
```

Comment:
What is `Units4Constant`? I've seen it in a number of places.

Reply:
the definition is three lines above